sass/layout/_legacy-sidebar-left.scss

```diff
@@ -65,33 +65,6 @@
         }
     }
 
-    .sidebar__divider {
```

Comment:
There's nothing in the app with the `sidebar__divider` CSS class applied to it, so this is presumably just old code.